### PR TITLE
refactor(utils): use @ctrl/tinycolor instead of custom color.ts

### PR DESCRIPTION
**Pull request by YunYouJun**
@ctrl/tinycolor is a nice color lib, and used by lots of ui lib.

We can import it to replace custom color uitil. At the same time, this means that we have more room to operate color in the future.

It is also possible to replace color-picker.
---

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by iamkun — APPROVED** [review]
LGTM

**Comment by imswk** [comment]
收到

**Review by adaex — APPROVED** [review]

**Comment by imswk** [comment]
LGTM

File: packages/element-plus/package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "normalize-wheel-es": "^1.1.1"
   },
   "devDependencies": {
+    "@ctrl/tinycolor": "^3.4.0",
```
Comment (sxzz) [inline packages/element-plus/package.json:59]: We should move it to `dependencies` @YunYouJun

Comment (YunYouJun) [reply packages/element-plus/package.json:59]: Ok, i create https://github.com/element-plus/element-plus/pull/4842 for it.

**Pull request merged**